go/chat/storage/inbox.go

```diff
@@ -924,9 +924,18 @@ func (i *Inbox) NewMessage(ctx context.Context, uid gregor1.UID, vers chat1.Inbo
 		return err
 	}
 
+	// Check for a delete, if so just auto return a version mismatch to resync.
+	// The reason is it is tricky to update max messages/orange line msg in
+	// this case.
+	switch msg.GetMessageType() {
+	case chat1.MessageType_DELETE, chat1.MessageType_DELETEHISTORY:
```

Comment:
We can't do this, for a big team like keybasefriends it creates a thundering herd.